test/fixtures/transform.out.css

```diff
@@ -1 +1 @@
-p {color: blue}p[data-jsx="123"] {color: red;}p[data-jsx="123"]{color: red}*[data-jsx="123"] {color: blue;}[href="woot"][data-jsx="123"] {color: red;}p[data-jsx="123"] a[data-jsx="123"] span[data-jsx="123"] {color: red;}p[data-jsx="123"] span {background: blue}p[data-jsx="123"] a[title="'w ' '  t'"][data-jsx="123"] {margin: auto}p[data-jsx="123"] span:not(.test) {color: green}p[data-jsx="123"],h1[data-jsx="123"] {color: blue;-webkit-animation-name:a123hahaha;-moz-animation-name:a123hahaha;animation-name:a123hahaha;-webkit-animation-delay: 100ms;-moz-animation-delay: 100ms;animation-delay: 100ms;}p[data-jsx="123"]:hover {color: red;}p[data-jsx="123"]::before {color: red;}[data-jsx="123"]:hover {color: red;}[data-jsx="123"]::before {color: red;}[data-jsx="123"]:hover p[data-jsx="123"] {color: red;}p[data-jsx="123"] + a[data-jsx="123"] {color: red;}p[data-jsx="123"] ~ a[data-jsx="123"] {color: red;}p[data-jsx="123"] > a[data-jsx="123"] {color: red;}p[data-jsx="123"] >> a[data-jsx="123"] {color: red;}@-webkit-keyframes a123hahaha {from { top: 0 }to { top: 100 }}@-moz-keyframes a123hahaha {from { top: 0 }to { top: 100 }}@keyframes a123hahaha {from { top: 0 }to { top: 100 }}@media (min-width: 500px) {.test[data-jsx="123"] {color: red;}}.test[data-jsx="123"] {display: block;}
+p {color: blue}p[data-jsx="123"] {color: red;}p[data-jsx="123"]{color: red}*[data-jsx="123"] {color: blue;}[href="woot"][data-jsx="123"] {color: red;}p[data-jsx="123"] a[data-jsx="123"] span[data-jsx="123"] {color: red;}p[data-jsx="123"] span {background: blue}p[data-jsx="123"] a[title="'w ' '  t'"][data-jsx="123"] {margin: auto}p[data-jsx="123"] span:not(.test) {color: green}p[data-jsx="123"],h1[data-jsx="123"] {color: blue;-webkit-animation-name:a123hahaha;-moz-animation-name:a123hahaha;animation-name:a123hahaha;-webkit-animation-delay:100ms;-moz-animation-delay:100ms;animation-delay:100ms;}p[data-jsx="123"]:hover {color: red;}p[data-jsx="123"]::before {color: red;}[data-jsx="123"]:hover {color: red;}[data-jsx="123"]::before {color: red;}[data-jsx="123"]:hover p[data-jsx="123"] {color: red;}p[data-jsx="123"] + a[data-jsx="123"] {color: red;}p[data-jsx="123"] ~ a[data-jsx="123"] {color: red;}p[data-jsx="123"] > a[data-jsx="123"] {color: red;}p[data-jsx="123"] >> a[data-jsx="123"] {color: red;}@-webkit-keyframes a123hahaha {from { top: 0 }to { top: 100 }}@-moz-keyframes a123hahaha {from { top: 0 }to { top: 100 }}@keyframes a123hahaha {from { top: 0 }to { top: 100 }}@media (min-width: 500px) {.test[data-jsx="123"] {color: red;}}.test[data-jsx="123"] {display: block;}
```

Comment:
actual diff, `-webkit-animation-delay: ` -> `-webkit-animation-delay:` trimmed the space